lib/vector-buffers/src/test/common/messages.rs

```diff
@@ -90,7 +90,7 @@ impl FixedEncodable for Message {
     }
 }
 
-#[derive(Clone, Debug, PartialEq, Eq, PartialOrd, Ord)]
+#[derive(Clone, Copy, Debug, PartialEq, Eq, PartialOrd, Ord)]
 pub(crate) struct SizedRecord(pub u32);
```

Comment:
Unsure if this will work, but I think it might be a slightly tighter scope:

```suggestion
pub(crate) struct SizedRecord(pub(crate) u32);
```